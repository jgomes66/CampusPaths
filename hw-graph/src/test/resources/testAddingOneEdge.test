# Test Adding One Edge Between Two Nodes
# Testing to see if we can add an edge between two nodes

# Create a graph
CreateGraph newGraph

# Add two nodes to our graph
AddNode newGraph node1
AddNode newGraph node2

# Add edge connecting node1 and node2
AddEdge newGraph node1 node2 e1

# Print nodes in graph and outgoing edges of node1
ListNodes newGraph
ListChildren node1 from newGraph