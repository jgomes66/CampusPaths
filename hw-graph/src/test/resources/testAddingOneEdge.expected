# Test Adding One Edge Between Two Nodes
# Testing to see if we can add an edge between two nodes

# Create a graph
created graph newGraph

# Add two nodes to our graph
added node node1 to newGraph
added node node2 to newGraph

# Add edge connecting node1 and node2
added edge e1 from node1 to node2 in newGraph

# Print nodes in graph and outgoing edges of node1
newGraph contains: node1 node2
The children of node1 in newGraph are: node2(e1)